homeassistant/components/climate/const.py

```diff
@@ -56,6 +56,11 @@
 # Device is reacting to activity (e.g. movement sensors)
 PRESET_ACTIVITY = "activity"
 
+# Device in holday mode (Eid, Rosh Hashanah, etc)
+PRESET_HOLIDAY = "holiday"
+
+# Device is in alternate program mode
+PRESET_ALTERNATE = "alternate"
```

Comment:
Presets can be defined in each integration. Please move the definition from the climate integration to the radiotherm integration.

Reply:
@MartinHjelmare Changes made, committed, and tests passed. Let me know if there's anything else.